src/stories/buttons/buttons.style.js

```diff
@@ -1,77 +1,38 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 import theme from "../theme";
 
-export const StyledButton = styled.button`
-  border: 2px solid;
-  background-color: ${theme.primary};
-  color: white;
-  padding: 10px 20px;
-  text-align: center;
-  text-decoration: none;
+export const StyledButtonComponent = styled.button`
   display: inline-block;
-  font-size: 16px;
-  cursor: pointer;
-`;
+  font-size: 1.4rem;
+  font-weight: 600;
+  width: max-content;
+  border: 2px solid ;
 
-export const Dngr = styled.button`
-  font-family: "Rubik", sans-serif;
-  border: 2px solid;
-  background-color: ${theme.warning};
-  border-top-left-radius: 2px;
-  border-top-right-radius: 2px;
-  color: white;
-  padding: 10px 20px;
-  text-align: center;
-  text-decoration: none;
-  display: inline-block;
-  font-size: 16px;
-  cursor: pointer;
-  &:hover,
-  &:active {
-    background-color: aqua;
-    color: beige;
-  }
-`;
+  ${({ type, isOutline,border  }) => css`
+    background-color: ${isOutline ? "transparent" : theme?.[type]};
+    color: ${isOutline ? theme?.[type] : "white"};
 
-export const Primary = styled.button`
-  border: 2px solid;
-  background-color: ${theme.primary};
-  border-top-left-radius: 2px;
-  border-top-right-radius: 2px;
-  color: white;
-  padding: 10px 20px;
-  text-align: center;
-  text-decoration: none;
-  display: inline-block;
-  font-size: 16px;
-  cursor: pointer;
-`;
+    border-color: ${border ? "transparent" : theme?.[type]};
+  `}
 
-export const Secondary = styled.button`
-  border: 2px solid ${theme.white};
-  width: 100;
-  background-color: ${theme.white};
-  border-top-left-radius: 2px;
-  border-top-right-radius: 2px;
-  color: black;
   padding: 10px 20px;
   text-align: center;
   text-decoration: none;
-  display: inline-block;
   font-size: 16px;
```

Comment:
What's the reason for defining the font size twice in the same component?

Reply:
by mistakenly 😅